src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/model/LinkState.java

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.flowmonitoring.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class LinkState {
+
+    private long rttLatency;
+    private long rttTimestamp;
+    private long oneWayLatency;
+
+    /**
+     * Get most accurate latency for the link.
```

Comment:
I'd reword this, as I understand it's not about accuracy, but about the freshness of the metric

Reply:
Fixed.